pip_chill/pip_chill.py

```diff
@@ -93,5 +93,11 @@ def chill(show_all=False, no_chill=False):
                 dependencies[requirement.key].version = distributions.pop(
                     requirement.key
                 ).version
+    
+    # Package 'pip' is kinda special. If `show_all`` is True, we want to elevate it
+    # from a dependency to a distribution, even if it is a dependency of another package.
```

Comment:
I am not sure we should. Previously we have discussed the possibility of adding a command line option to list "special" packages we always want listed. The same way, when you add, for instance, the Django REST Framework, it's assumed you also want a Django for it.

I'm in doubt whether this behavior is desired - if I want package B which depends on A, in which cases A is special enough to warrant its inclusion, even though B specifies it.

Reply:
reverted 👍

Reply:
Hmm... 🤔 I'll revert it here (since, tbh, that's not really what this PR is about) and make another issue describing exactly the behaviour i found and why i've originally added this.